client/src/actions/practice.ts

```diff
@@ -0,0 +1,46 @@
+import 'whatwg-fetch';
```

Comment:
We should be able to work around this... just put a TODO and I'll look at it later.